dataikuapi/dssclient.py

```diff
@@ -629,6 +629,59 @@ def create_global_api_key(self, label=None, description=None, admin=False):
         key = resp.get('key', '')
         return DSSGlobalApiKey(self, key)
 
+    def delete_global_api_key(self, key):
+        """
+        Delete a specific global API Key
+
+        Note: this call requires an API key with admin rights
+        """
+        resp = self._perform_http(
+            "DELETE", "/admin/globalAPIKeys/" + key)
+        if resp.status_code != 200:
+            raise Exception('API key creation failed : %s' % resp.json())
+
+    ########################################################
+    # Personal API Keys
+    ########################################################
+
+    def list_personal_api_keys(self):
+        """
+        List all personal API keys:
+            - not admin: only the keys belonging to the owner of the key
+            - admin: All the personal keys
+
+        :returns: All personal API keys, as a list of dicts
+        """
+        return self._perform_json(
+            "GET", "/personalAPIKeys/")
+
+    def get_personal_api_key(self, key):
+        """
+        Get a specific API key from the user doing the request
+
+        :param str key: the secret key of the desired API key
+        :returns: A :class:`dataikuapi.dss.admin.DSSPersonalApiKey` API key handle
+        """
+        return DSSPersonalApiKey(self, key)
+
+    def create_personal_api_key(self, label):
```

Comment:
`label` should not be a positional argument given it is not required

Reply:
This is to be retro-compatible with this function deleted line 964

```
    def create_personal_api_key(self, label):
```

(Initially I didn't have the label parameter but did it that way so it doesn't brake existing scripts)

Reply:
It shouldn't be an issue, `label` could be called either by doing `create_personal_api_key(label="my label")` or `create_personal_api_key("my label")`